app/serializers/api/beta/commodity_serializer.rb

```diff
@@ -6,7 +6,17 @@ class CommoditySerializer < GoodsNomenclatureSerializer
       attributes :chapter_description,
                  :chapter_id,
                  :heading_description,
-                 :heading_id
+                 :heading_id,
+                 :validity_start_date,
+                 :validity_end_date
+
+      def validity_start_date
+        super&.iso8601
+      end
+
+      def validity_end_date
+        super&.iso8601
+      end
     end
```

Comment:
I think we should update the relevant spec:
`spec/serializers/api/beta/commodity_serializer_spec.rb`